packages/core/src/exchanges/compose.test.ts

```diff
@@ -1,14 +1,11 @@
 import { empty, Source } from 'wonka';
 import { Exchange } from '../types';
 import { composeExchanges } from './compose';
+import { noop } from '../utils';
 
-const mockClient = {
-  debugTarget: {
-    dispatchEvent: jest.fn(),
-  },
-} as any;
-const forward = jest.fn();
+const mockClient = {} as any;
```

Comment:
@kitten if we're going ahead with not exposing the dispatchDebug can we go ahead and flag it as private rather than just changing our consumption of the client?